### PR TITLE
add blinking for pauses

### LINKED ISSUE #273: Implement animation for long pauses
**Is this related to a problem? Please describe.**
If the audio and text file have a pause (added via Kukarella voice effects), then the character will stay still with eyes open for the entirety of the pause.

**Describe the solution you'd like**

- identify long frames
- set the character's body position to the default standing position
- break the long paused frame into multiple smaller frames
- implement blinking
- implement eyes moving left and right (*if there's time to implement*)

### PR DESCRIPTION
**Pull request by vnajack**
- for pauses that are longer than 2 seconds, I added blinking and a smooth transition period for the body to go to the relaxed pose
- added an "idle" state, so more can be added to the idle animation in future
- closes #273

### CODE REVIEW

File: speaking-portal-project/src/types.ts [hunk 1]
```diff
@@ -157,14 +157,24 @@ export class Avatar {
...
             let n = Math.random()
             if ((n <= 0.3 && currentSec % 4 == 0) || (n > 0.3 && n <= 0.6 && currentSec % 3 == 0)) {
                 this.eyes.close(currentSec)
             }
         }
     }
 
+    // if character is idle (e.g., during a long pause), then the body position will be relaxed
+    idleBody(currentSec: number): void{
+        // TODO: decide if we want to implement breathing animation here
```
Comment (LoganParker) [inline speaking-portal-project/src/types.ts]: I say no to the breathing, I think the current actions we have are pretty good

**Review by LoganParker — APPROVED** [review]
This all looks great Veronica!! Thank you for doing all of this, the idle animation looks a lot better now :)

File: speaking-portal-project/src/types.ts (hunk 1 above)
Comment (vnajack) [inline speaking-portal-project/src/types.ts]:
```suggestion
        // Can add different idle body animation here, if desired
```

**Review by edouarde1 — APPROVED** [review]
Looks great thanks Veronica :)

**Review by mkuelker — APPROVED** [review]
Solid Work!

**Pull request merged**